webapp/components/Comment/Comment.vue

```diff
@@ -141,6 +151,10 @@ export default {
     },
   },
   methods: {
+    reply() {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-10T19:02:29Z" title="Friday, January 10th 2020, 8:02:29 pm +01:00">Jan 10, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -168,6 +176,23 @@ export default {
         this.$toast.error(err.message)
       }
     },
+    answerComment() {
+      const slug =
+        '<a class="mention" href="/profile/' +
+        this.comment.author.id +
+        '" data-mention-id="' +
+        this.comment.author.slug +
+        '" target="_blank" contenteditable="false">@d' +
+        this.comment.author.slug +
+        '</a>'
+      document.querySelector('.editor-content div').focus()
```


Don't use globals like `document` which are not available in a NodeJS environment.

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-01-14T17:42:29Z" title="Tuesday, January 14th 2020, 6:42:29 pm +01:00">Jan 14, 2020</time>_

---


wir haben vue-scroll over nuxt in the system 
you can ever on the element set: 
` v-scroll-to="'.editor'"`

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-16T11:09:06Z" title="Thursday, January 16th 2020, 12:09:06 pm +01:00">Jan 16, 2020</time>_

---


Scrolling != focus

You can scroll to an element programatically: https://github.com/rigor789/vue-scrollto#programmatically with `this.$scrollTo(element, duration, options)`.